include/ppx/grfx/grfx_descriptor.h

```diff
@@ -31,22 +31,25 @@ namespace grfx {
 //!
 struct DescriptorBinding
 {
-    uint32_t              binding         = PPX_VALUE_IGNORED;               //
-    grfx::DescriptorType  type            = grfx::DESCRIPTOR_TYPE_UNDEFINED; //
-    uint32_t              arrayCount      = 1;                               // WARNING: Not VkDescriptorSetLayoutBinding::descriptorCount
-    grfx::ShaderStageBits shaderVisiblity = grfx::SHADER_STAGE_ALL;          // Single value not set of flags (see note above)
+    uint32_t               binding         = PPX_VALUE_IGNORED;               //
+    grfx::DescriptorType   type            = grfx::DESCRIPTOR_TYPE_UNDEFINED; //
+    uint32_t               arrayCount      = 1;                               // WARNING: Not VkDescriptorSetLayoutBinding::descriptorCount
+    grfx::ShaderStageBits  shaderVisiblity = grfx::SHADER_STAGE_ALL;          // Single value not set of flags (see note above)
+    DescriptorBindingFlags flags;
```

Comment:
I think we should check the flags in the DX12 Create and emit a warning or error if these are set? Although I don't know how DX12 equivalent of descriptors behave, perhaps partial bindings are always supported.

Reply:
Ahhh - my mistake, I did add a function for querying for support, seems I forgot to apply it. Will follow-up on this.